### PR TITLE
remove the legacy option

### PR DESCRIPTION
**Pull request by cirrusasf**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.2]
+
+### Modified
+* the dems.md and rtc_product_guide.md to remove the legacy DEM option in the HyP3 processing for RTC
```
Comment (AndrewPlayer3) [inline CHANGELOG.md:12]:
```suggestion
* the RTC and DEM guides to inform users that the legacy DEM options are no longer available
```

File: docs/guides/rtc_product_guide.md [hunk 2]
```diff
@@ -54,15 +54,9 @@ The RTC product package includes a Layover-Shadow mask (see [Image Files section
...
-
-!!! tip "Deprecation of Legacy DEMs for RTC Processing"
-
-    We are considering eliminating the option to use our legacy DEM dataset (NED/SRTM) as a HyP3 processing option for RTC. We would value your feedback as we decide if we will make this change. How would you be impacted if the NED/SRTM DEM option was no longer available? Would it affect your current workflows? 
-
-    Please send your feedback to [uso@asf.alaska.edu](mailto:uso@asf.alaska.edu).
+Users do not have the option to use the legacy DEMs anymore when processing RTC jobs [On Demand in Vertex](https://search.asf.alaska.edu/#/?topic=onDemand "Vertex On Demand Documentation" ){target=_blank} and when using the [API](https://hyp3-docs.asf.alaska.edu/using/api/ "https://hyp3-docs.asf.alaska.edu/using/api" ){target=_blank} or [SDK](https://hyp3-docs.asf.alaska.edu/using/sdk/ "https://hyp3-docs.asf.alaska.edu/using/sdk" ){target=_blank}.
```
Comment (AndrewPlayer3) [inline docs/guides/rtc_product_guide.md:59]:
```suggestion
Users no longer have the option to use legacy DEMs when processing RTC jobs [On Demand in Vertex](https://search.asf.alaska.edu/#/?topic=onDemand "Vertex On Demand Documentation" ){target=_blank} or when using the [API](https://hyp3-docs.asf.alaska.edu/using/api/ "https://hyp3-docs.asf.alaska.edu/using/api" ){target=_blank} or [SDK](https://hyp3-docs.asf.alaska.edu/using/sdk/ "https://hyp3-docs.asf.alaska.edu/using/sdk" ){target=_blank}.
```

File: CHANGELOG.md (hunk 1 above)
Comment (cirrusasf) [reply CHANGELOG.md:12]: DEM is not guide. How about " the DEM description and RTC product guide to inform users that the legacy DEM options are no longer available"

Comment (AndrewPlayer3) [reply CHANGELOG.md]: That works too.

**Review by AndrewPlayer3 — APPROVED** [review]

**Pull request merged**